renovate.json

```diff
@@ -14,5 +14,13 @@
   "suppressNotifications": [ "artifactErrors" ],
   "labels": [
     "dependency"
+  ],
+  "regexManagers": [
```

Comment:
Not sure how well this plays with the component template, but probably good enough for now.